packages/webapp-libs/webapp-tenants/src/providers/currentTenantProvider/currentTenantProvider.component.tsx

```diff
@@ -22,9 +24,20 @@ export type TenantPathParams = {
 export const CurrentTenantProvider = ({ children }: CurrentTenantProviderProps) => {
   const params = useParams<TenantPathParams>();
   let { tenantId } = params;
+  const { data } = useCommonQuery();
+  const userId = (data?.currentUser as CurrentUserType)?.id;
```

Comment:
is it possible to adjust the type of this currentUser so we don't need to use this casting? I'm not quite sure why is it necessary